bench-tps/src/bench.rs

```diff
@@ -702,13 +793,24 @@ fn nonced_transfer_with_padding(
             TRANSFER_TRANSACTION_LOADED_ACCOUNTS_DATA_SIZE,
         ),
     ]);
-    let message = Message::new_with_nonce(
-        instructions,
-        Some(&from_pubkey),
-        nonce_account,
-        &nonce_authority.pubkey(),
+    let versioned_nonce_transaction = VersionedTransaction::try_new(
+        VersionedMessage::Legacy(Message::new_with_nonce(
+            instructions,
+            Some(&from_pubkey),
+            nonce_account,
+            &nonce_authority.pubkey(),
+        )),
+        &[from_keypair],
+    )
+    .unwrap();
+
+    info!(
+        "==== versioned_nonce_transaction {:?}",
+        versioned_nonce_transaction
     );
-    Transaction::new(&[from_keypair, nonce_authority], message, nonce_hash)
+
+    // TODO - need to sign the tx with `nonce_hash`?
```

Comment:
could set the blockhash using `VersionedMessage::set_recent_blockhash`?

the `new_with_nonce` explicitly sets it to default. I'm not sure what the expected behavior is.